src/framework/global/settings.cpp

```diff
@@ -297,9 +301,12 @@ void Settings::beginTransaction(bool notifyToOtherInstances)
     m_localSettings = m_items;
     m_isTransactionStarted = true;
 
+    UNUSED(notifyToOtherInstances)
```

Comment:
should be in 'else' block?

Reply:
I don't think it's important... but the code is a little cleaner.